analysis_options.yaml

```diff
@@ -1,4 +1,2 @@
 include: package:pedantic/analysis_options.1.9.0.yaml
-analyzer:
-  enable-experiment:
-  - non-nullable
```

Comment:
is there a reason this was removed?

Reply:
It was considered an experimental flutter feature in the previous versions of flutter but isn't in the latest v3 of flutter